main.go

```diff
@@ -67,6 +69,18 @@ func doMain(args []string) {
 
 			Action: cmdServer,
 		},
+		{
+			Name:  "migrate",
+			Usage: "Run migrations and exit",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "tenant",
+					Usage: "Tenant ID (optional).",
```

Comment:
if I get this correctly `./deviceauth migrate` will migrate all dbs, `./deviceauth migrate --tenanant 123` just a single one right ?

Reply:
No, `migrate` will migrate the default DB, `migrate --tenant 123` will migrate only the DB owned by tenant 123.